docs/docsource/reference/bucket_ingestion_metrics/get_all_metrics.rst

```diff
@@ -0,0 +1,40 @@
+GET All Metrics	
+===============	
+
+This request retrieves all metrics for all S3 Connector metadata	
+ingestion locations. Zenko returns three categories of information	
+(metrics) about system operations: completions, throughput, and 	
+pending operations. Completions are returned for the preceding 24	
+hours, throughput for the preceding 15 minutes, and pending    
+transactions are returned as a simple aggregate.    
+
+**Endpoint**	 
+
+ .. code::
+
+   /_/metrics/ingestion/all	
```

Comment:
should these routes also start with `backbeat/api` like the other routes?

Reply:
OK.